src/index.html

```diff
@@ -5,9 +5,47 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+     href="https://fonts.gstatic.com"
+     crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section class="card" data-qa="card">
+      <img
+        class="card__img"
+        src="./images/imac.jpeg"
+        alt="imac"
+      >
+      <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <div class="card__code">Product code: 195434</div>
+
+      <div class="card__reviews">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="card__reviews-count">Reviews: 5</div>
+      </div>
+
+      <div class="card__price">
+        <div class="card__price-title">
+          Price:
+        </div>
+        <div class="card__price-price">
+          $2,199
+        </div>
+      </div>
+
+      <div class="card__buy-btn">
+        <a data-qa="hover" class="card__buy-btn-link">Buy</a>
+      </div>
+
+    </section>
```

Comment:
```suggestion
      </div>
    </section>
```
Don't leave empty lines between parent and child elements